src/Hedgehog/Gen.fs

```diff
@@ -502,15 +502,20 @@ module Gen =
         toRandom g
         |> Random.run seed 30
 
-    let printSample (g : Gen<'a>) : unit =
-        let forest = sampleTree 10 5 g
-        for tree in forest do
-            printfn "=== Outcome ==="
-            printfn "%A" (Tree.outcome tree)
-            printfn "=== Shrinks ==="
-            for shrink in Tree.shrinks tree do
-                printfn "%A" (Tree.outcome shrink)
-            printfn "."
+    let formatSample (gen : Gen<'a>) : string =
+        String.concat Environment.NewLine [
+            let forest = sampleTree 10 5 gen
+            for tree in forest do
+                yield "=== Outcome ==="
+                yield sprintf "%A" (Tree.outcome tree)
+                yield "=== Shrinks ==="
+                for shrink in Tree.shrinks tree do
+                    yield sprintf "%A" (Tree.outcome shrink)
+                yield "."
+        ]
+
+    let printSample (gen : Gen<'a>) : unit =
```

Comment:
I think we should drop the `print*` family of functions

Reply:
@moodmosaic Thoughts on this?

Reply:
I believe we should, yes. After all they're one liners e.g. `printfn "%s" (render gen)` (btw, it's still formatSample in the diff, I thought we're going with `render` instead?)

Reply:
Edit: Now I see, `render` is on the property level.

Reply:
https://github.com/hedgehogqa/fsharp-hedgehog/pull/274/files#r576824359

Reply:
Dropped the `print*` variations, and renamed the missed functions to `render*`.

Do we want `Gen.renderSample` to just be `Gen.render`?

Reply:
I don't think so.  `Sample` has meaning here.  It is not the entire `Gen`<_>` that is being rendered.  Just a few samples (5 or 10, I forgot which argument is which).